packages/clerk-js/src/ui/components/AccountlessPrompt/ClerkLogoIcon.tsx

```diff
@@ -0,0 +1,236 @@
+export function ClerkLogoIcon() {
```

Comment:
🔧  Can we move this icon into the `/ui/icons` folder as a `.svg` file and then export it from the `/ui/index.ts` file as we do with all of our icons?

Reply:
These are custom icons specifically used for the Keyless toast, and not really intended to be used anywhere else. I added comments above the icons to make this clear!